avm_scripts/test-example.sh

```diff
@@ -28,7 +28,8 @@ fi
 echo "==> go test"
 echo $TEST_TIMEOUT
 terraform version
-go test -v -timeout=$TEST_TIMEOUT -run TestExample --
+test_failed=false
+go test -v -timeout=$TEST_TIMEOUT -run TestExample || test_failed=true
```

Comment:
Will go test return a none 0 exit code if it fails? Is that how this will work?

Reply:
Yes, once `go test` failed we'll set `test_failed` to `true`, then return `1` finally.